docs/installation/mac.md

```diff
@@ -107,16 +107,26 @@ and choosing "Open" from the pop-up menu.
 
 7. Press "Close" to exit.
 
+## Learn about your Toolbox installation
```

Comment:
I'm not sure that works as well. What is the first thing users want to do after installing?  I think they'd rather do something than learn about the installation itself --- which is why that comes after.